### ISSUE TITLE
Very Nice View

### ISSUE THREAD

**Issue by xu6148152** [hanks-zyh/HTextView#10]
How about using List<Float> instead of float[] with gap? because I think that the size of array may be an potential issue when the size of text is longer than 100.

**Comment by hanks-zyh** [comment]
@xu6148152  I agree with you，it‘s a bug，but now the HTextView can only show one line, it should be use to show a short text, so I think the length will not more that 100, but what you said is a better that what i did.

**Comment by sjtu-lyj** [comment]
@xu6148152  is there any way to show mutiple lines?
